watson/cli.py

```diff
@@ -99,24 +99,32 @@ def help(ctx, command):
     click.echo(cmd.get_help(ctx))
 
 
-def _start(watson, project, tags, restart=False):
+def _start(watson, project, tags, restart=False, from_=None):
     """
     Start project with given list of tags and save status.
     """
     current = watson.start(project, tags, restart=restart)
+    if from_:
+        # modify the current frame with optional start time instead of "now"
+        watson.current = dict(start=from_, project=project, tags=tags)
+        current = watson.current
```

Comment:
Why not choosing to hack around the `watson.start` method?

Reply:
Why add more code and complexity if same result can be achieved with more elegant solution? Of course the elegance is in the eye of the beholder (and committer in this case :))